nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -91,6 +91,34 @@ protected void setupArgline() {
 
     }
 
+    @Override
+    protected Xpp3Dom applyNonDexConfig(Xpp3Dom configuration) {
+        return addExcludedGroups(super.applyNonDexConfig(configuration));
+    }
+
+    private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
+        for (Xpp3Dom config : configNode.getChildren()) {
+            if ("excludedGroups".equals(config.getName())) {
+                Logger.getGlobal().log(Level.INFO, "Adding excluded groups to existing ones");
+                String current = config.getValue();
+                current = "," + current;
+                // It seems there is an error if you have the variable
```

Comment:
Please add an integration test consisting of a multi module maven project; the project should define excluded groups both in the parent pom, and in the submodules. The tests should fail if they are not properly excluded. Then add also some tests in each module that have our NonDex Ignore. I think that because I remove the ${excludeGroups} I may be breaking multimodule projects, if the excluded groups defined in the parent pom are inherited through the variable I am removing.